partuniverse/partsmanagement/models.py

```diff
@@ -882,6 +883,9 @@ def save(self, *args, **kwargs):
             if not self.id:
                 # We got a new Transaction
                 storageitem = StorageItem.objects.get(pk=self.storage_item.id)
+                if storageitem.on_stock + Decimal(self.amount) < 0:
```

Comment:
This will fail, when a storage item is freshly created or not having a stock assinged at all

Reply:
Did I add the testcase correctly?
Any suggestions for the below zero check, that will not fail?